bin/generate-post-meta-image

```diff
@@ -1,34 +1,68 @@
 #!/bin/sh -e
 
-text="$1"
-bg="$2"
-out=$3
+tmp=$(mktemp -d)
+bg="#2421AB"
 res=2160x1080
 
-tmp=$(mktemp -d)
+while [[ $# -gt 0 ]] && [[ "$1" == "--"* ]]; do
+  case "$1" in
+    "--resolution" | "-r")
+      res=$2
+      shift
+      shift
+      ;;
 
-if [[ "$bg" == "" ]]; then
-  bg="#2421AB"
-  convert -size $res xc:$bg $tmp/bg.png
-else
-  convert $2 -resize $res^ $tmp/bg-uncropped.png
-  convert $tmp/bg-uncropped.png -gravity center -crop $res+0+0 -fill black -colorize 40% $tmp/bg.png
-fi
+    "--text" | "-t")
+      text=$2
+      shift
+      shift
+      ;;
 
-convert -size $res xc:"#00000000" \
-  -pointsize 100 -fill white -draw 'text 100,160 "Blog Post" ' \
-  $tmp/text-top.png
+    "--bg" | "-b")
```

Comment:
Shouldn't the long version be `background`?

Reply:
indeed. fixed